test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -261,22 +261,25 @@ void ExpressionEvaluator::operator()(Identifier const& _identifier)
 
 void ExpressionEvaluator::operator()(FunctionCall const& _funCall)
 {
-	evaluateArgs(_funCall.arguments);
+	vector<optional<LiteralKind>> literalArguments;
+	if (BuiltinFunction const* builtin = m_dialect.builtin(_funCall.functionName.name))
+		literalArguments = builtin->literalArguments;
```

Comment:
Copying the vector seems a bit wasteful here, but I guess it's fine.

Reply:
I can actually change it to a pointer and make it a nullptr already if the vector is empty.

Reply:
Both fine!